### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by diegoh40**
- Implement a connection between a Ruby on Rails back-end and React front-end.
- Understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.

### CODE REVIEW

**Review by rustam-tolipov — COMMENTED** [review]
Hi @diegoh40, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Congratulations on the effort you've put into this project! We appreciate it and know that great progress has been made, however, we need to make a small adjustment before we can move forward with the review. Please remove or replace the Microverse logo from your readme file if you have a personal logo available for use. 

Please note that all projects must include a Future Features section outlining 1-3 potential features to be added.

Once these steps are completed, we can move forward with the review process. Thank you for your understanding!


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/javascript/controllers/components/App.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+function App () {
+  return (
+    <div>My app is awesome!</div>
```
Comment (Ismailco) [inline app/javascript/controllers/components/App.js]: - [x] The App component should include routes from `react-router`, kindly make sure to add this so the app is working properly

Comment (Ismailco) [inline app/javascript/controllers/components/App.js]: -[ ] Your components folder should include a treating component, kindly make sure to implement this so your app has a greeting component to render the random message

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @diegoh40, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [ ] Kindly make sure to create a store, an action , and a reducer that uses the url to the endpoint from the Rails API

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
the error was corrected and updated, I add the screenshot of my browser:

image

File: app/javascript/controllers/components/App.js (hunk 1 above)
Comment (diegoh40) [reply app/javascript/controllers/components/App.js]: done

image

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @diegoh40, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**